habitat-baselines/habitat_baselines/config/objectnav/ddppo_objectnav_hm3d_stretch_continuous_action.yaml

```diff
@@ -0,0 +1,84 @@
+# @package _global_
+
+defaults:
+  - /habitat_baselines: habitat_baselines_rl_config_base
+  - /benchmark/nav/objectnav: objectnav_hm3d_stretch_velocity_controller
+  - /habitat_baselines/rl/policy/obs_transforms:
+    - center_cropper_base
+    - resize_shortest_edge_base
+  - _self_
+
+habitat:
+  environment:
+    iterator_options:
+      max_scene_repeat_steps: 50000
+  task:
+    success_reward: 2.5
+    slack_reward: -1e-3
+
+habitat_baselines:
+  torch_gpu_id: 0
+  tensorboard_dir: "tb"
+  video_dir: "video_dir"
+  test_episode_count: -1
+  eval_ckpt_path_dir: "data/new_checkpoints"
+  num_environments: 4
+  checkpoint_folder: "data/new_checkpoints"
+  trainer_name: "ddppo"
```

Comment:
why not VER?

Reply:
People are free to use VER if they want. I just followed the convention that this folder had